src/features/comments.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { createComment, deleteComment, getPostComments } from '../api/comments';
+import { Comment } from '../types/Comment';
+
+type Comments = {
+  comments: Comment[];
+  isLoading: boolean;
+  error: string;
+};
+
+const initialState: Comments = {
+  comments: [],
+  isLoading: false,
+  error: '',
+};
+
+export const loadComments = createAsyncThunk(
+  'comments/fetch',
+  async (postId: number) => {
+    return getPostComments(postId);
+  },
+);
+
+export const addNewComment = createAsyncThunk(
+  'comments/addComment',
+  async (comment: Omit<Comment, 'id'>) => {
+    return createComment(comment);
+  },
+);
+
+export const removeComment = createAsyncThunk(
+  'comments/removeComment',
+  (commentId: number) => {
+    deleteComment(commentId);
+
+    return commentId;
+  },
+);
+
+const commentsSlice = createSlice({
+  name: 'comments',
+  initialState,
+  reducers: {},
+  extraReducers: builder => {
+    builder.addCase(loadComments.pending, state => {
+      state.isLoading = true;
+    });
+
+    builder.addCase(
+      loadComments.fulfilled,
+      (state, action: PayloadAction<Comment[]>) => {
+        state.comments = action.payload;
+        state.isLoading = false;
+      },
+    );
+
+    builder.addCase(loadComments.rejected, state => {
+      state.error = 'Error';
+      state.isLoading = false;
+    });
+
+    builder.addCase(
+      addNewComment.fulfilled,
+      (state, action: PayloadAction<Comment>) => {
+        state.comments.push(action.payload);
+      },
+    );
+
+    builder.addCase(addNewComment.rejected, state => {
+      state.error = 'error';
```

Comment:
Similar to the previous comment, consider using the error information from the rejected action to set a more informative error message.